configure.ac

```diff
@@ -585,30 +585,48 @@ AC_CHECK_FUNCS(scandir)
 AC_CHECK_FUNCS(fork waitpid execv pipe,[enable_xcmd=yes],[enable_xcmd=no])
 AM_CONDITIONAL([ENABLE_XCMD], [test "xyes" = "x$enable_xcmd"])
 
+AC_ARG_ENABLE([xml],
+	[AS_HELP_STRING([--enable-xml],
+		[enable xml support [no]])])
+
 AH_TEMPLATE([HAVE_LIBXML],
 	[Define this value if libxml is available.])
 dnl About the condition of version
 dnl see https://mail.gnome.org/archives/xml/2010-February/msg00008.html
-PKG_CHECK_MODULES(LIBXML, [libxml-2.0 >= 2.7.7],
-		       [have_libxml=yes
-		       AC_DEFINE(HAVE_LIBXML)],
-		       [have_libxml=no])
+AS_IF([test "${enable_xml}" = "yes"], [
```

Comment:
Contributors of ctags will link more libraries to ctags in the future.
So could you add comments to configure.ac about "avoiding automagic dependencies"?
So the contributors may do the same as you did in this pull request like:

```
dnl This enforces explicit feature usage regardless of the libraries
dnl available on the build system. This avoids automagic dependencies which
dnl can cause issues for source-based distros [1].
dnl [1]: https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Automagic_dependencies
```